### PR TITLE
added-files

### PR DESCRIPTION
**Pull request by shmielminz**
Hope all is good.

### CODE REVIEW

File: database/1 table.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	CustomerFirstName varchar(25) not null constraint ck_AttractionTruck_Customer_FirstName_cannot_be_blank check(CustomerFirstName > ''),
+	CustomerLastName varchar(25) not null constraint ck_AttractionTruck_Customer_LastName_cannot_be_blank check(CustomerLastName > ''),
+	CustomerAddress varchar(100) not null constraint ck_AttractionTruck_Customer_Address_cannot_be_blank check(CustomerAddress > ''),
+	Country varchar(13) not null constraint ck_AttractionTruck_Country_must_be_in_Canada_UnitedStates_and_Mexico check(Country in('Canada', 'United States', 'Mexico')),
+	CustomerPhoneNum varchar(12) not null constraint ck_AttractionTruck_Customer_phone_num_must_be_valid check(replace(replace(replace(replace(CustomerPhoneNum,')',''),'(',''),' ',''),'-','') like '[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]'),
+	DeliveryDate date not null constraint ck_AttractionTruck_DeliveryDate_must_be_after_2023 check(year(DeliveryDate) >= 2023),
+	PickupDate date not null,
+	PricePerDay int not null constraint ck_AttractionTruck_PricePerDay_must_be_between_50_and_10000 check(PricePerDay between 50 and 10000),
```
Comment (YFisch) [inline database/1 table.sql]: For the price wouldn't it be better a decimal?

File: database/1 table.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	IsDamaged bit,
+	DamageDesc varchar(200) not null default '',
+	DamagePrice int constraint ck_AttractionTruck_DamagePrice_must_be_between_1_and_100000 check(DamagePrice between 1 and 100000),
```
Comment (YFisch) [inline database/1 table.sql]: Here it's the same as above For the price wouldn't it be better a decimal?

**Review by YFisch — CHANGES_REQUESTED** [review]
I reviewed the work, everything seems to be good, I only have two small comments.

**Comment by shmielminz** [comment]
Sorry, silly mistake.

File: database/1 table.sql (hunk 1 above)
Comment (YFisch) [reply database/1 table.sql]: all good

File: database/1 table.sql (hunk 2 above)
Comment (YFisch) [reply database/1 table.sql]: all good

**Pull request closed without merging**